adoc/chapters/programming_interface.adoc

```diff
@@ -14373,6 +14373,12 @@ functions and common member functions that are listed in
 <<sec:reference-semantics>> in <<table.specialmembers.common.reference>> and
 <<table.hiddenfriends.common.reference>>, respectively.
 
+As with all SYCL objects that have the common reference semantics, kernel
+identifiers are equality comparable.  Two [code]#kernel_id# objects that refer
+to the same application kernel or to the same device built-in kernel are
+guaranteed to compare equal.  Two [code]#kernel_id# objects that refer to
+different kernels are guaranteed to compare unequal
```

Comment:
```suggestion
different kernels are guaranteed to compare unequal.
```

Reply:
Perhaps we could condense this wording by using a mathematical `if and only if` instead?

Reply:
Good idea.  Wording simplified in 3086fb3d52186a319cbc998ab0bb3f639fc72955.